legacy/expressions/visitor.go

```diff
@@ -210,7 +210,7 @@ func (v *legacyVisitor) VisitAdditionOrSubtraction(ctx *gen.AdditionOrSubtractio
 		// we are adding a datetime and a time
 
 		// create expression which converts arg2 to minutes
-		asMinutes := fmt.Sprintf(`format_time(%[1]s, "h") * 60 + format_time(%[1]s, "m")`, arg2)
+		asMinutes := fmt.Sprintf(`format_time(%[1]s, "tt") * 60 + format_time(%[1]s, "m")`, arg2)
```

Comment:
curiously golang date formatting doesn't support away to get a non-zero padded 24 hour hour-of-day value

Reply:
Seems this is a bug in `FormatTime` then as well no? Or are we removing the option to use `h`? Seems we need to support it but we'll need to do it ourselves.

Reply:
`FormatTime` currently supports `h`, `hh`, and `tt` but not `t` (which is also not listed as an option in the docs). Agree we should find a way to add support for `t` tho.

Reply:
https://github.com/nyaruka/goflow/issues/569